README.md

```diff
@@ -153,6 +154,7 @@ This plugin creates device plugin endpoints based on the configurations given in
         },
         {
             "resourceName": "intel_sriov_dpdk",
+            "isRdma": true,
```

Comment:
Shall we have a separate resourceName example here for Mellanox rdma devices?
The current one seems confusing as Intel NIC doesn't support rdma, right?

Reply:
absolutely this example is misleading we should have on for intel and one for mellanox